bindings/pyroot/src/TMemoryRegulator.cxx

```diff
@@ -191,6 +191,29 @@ void PyROOT::TMemoryRegulator::RecursiveRemove( TObject* object )
    }
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// clean up all tracked objects
+
+void PyROOT::TMemoryRegulator::RecursiveRemoveAll()
```

Comment:
Minor comment.  The name is a little bit misleading at it sounds like it force a RecursiveRemove call even for the object that explicitly disabled it for themselves ... which is not the case (nor what we want :) ).   Maybe just name this 'Clear' (or something else) as in other ROOT collection this method delete the content its owns.